src/main/scala/com/databricks/spark/avro/AvroSaver.scala

```diff
@@ -44,24 +44,31 @@ import org.apache.hadoop.mapred.JobConf
  */
 object AvroSaver {
 
-  def save(dataFrame: DataFrame, location: String): Unit = {
+  def save(
+      dataFrame: DataFrame,
+      location: String,
+      recordName: String = "topLevelRecord",
+      recordNamespace: String = ""): Unit = {
```

Comment:
Can you please change save to take an optional `Map[String, String] parameters` and get `recordNamespace` and `recordName` from there. In case there are more optional parameters in the future, it would be nicer to pass them this way.